### PR TITLE
Added bettervannillabuilding

### PR DESCRIPTION
**Pull request by mrhappy200**
BetterVanillaBuilding is a texturepack that uses optifine connected textures in a way like never seen before. Giving its users access to a big range of new building blocks, as well as new block variations in biomes to explore, new mobs to look at and new armors and tools to wield!

### CODE REVIEW

**Comment by g-plane** [comment]
Description is too long. Can you shorten it?

**Comment by mrhappy200** [comment]
Of course, is this better?

File: readme.md [hunk 1]
```diff
@@ -179,7 +179,7 @@
 - [Cyber Optics](https://www.planetminecraft.com/texture-pack/cyber-optics-v10/) - Sci-Fi resource pack.
 - [Soartex](https://soartex.net/) - Smooth resource pack with mod support.
 - [Faithful](https://faithful.team/) - Smooth textures for a furbished default look.
-- [BetterVanillaBuilding](https://www.curseforge.com/minecraft/texture-packs/bettervanillabuilding) - "BetterVanillaBuilding is a texturepack that uses optifine connected textures in a way like never seen before. Giving its users access to a big range of new building blocks, as well as new block variations in biomes to explore, new mobs to look at and new armors and tools to wield!"
+- [BetterVanillaBuilding](https://www.curseforge.com/minecraft/texture-packs/bettervanillabuilding) - "BetterVanillaBuilding is a recource pack that adds loads of new blocks with connected textures in optifine"
```
Comment (g-plane) [inline readme.md:182]:
```suggestion
- [BetterVanillaBuilding](https://www.curseforge.com/minecraft/texture-packs/bettervanillabuilding) - Adds loads of new blocks with connected textures in optifine.
```

**Review by g-plane — APPROVED** [review]
Thanks!

**Pull request merged**